scripts/repoHandlers.js

```diff
@@ -72,40 +101,42 @@ function updateTrackedReposInBitbucketOrg(trackedRepos, websiteUrl) {
  * @param {boolean} isOrg - Whether github organization or user.
  */
 function updateTrackedReposInGitHub(trackedRepos, websiteUrl, ownerType) {
-	const allRepo = ownerType == 'org' ? document.getElementById('org-repositories') : document.getElementById('user-repositories-list');
-	const repoUrl = Array.from(allRepo.querySelectorAll('a[itemprop="name codeRepository"]'));
-
-
-	repoUrl.forEach((item) => {
-		const link = item.getAttribute('href').split('/');
-		const repoName = link[link.length - 1];
-
+	// Check if selectors for new UI exist
+	const newUI = document.querySelector('[data-testid="list-view-items"]') ? true : false
```

Comment:
```suggestion
	const newUI = !!document.querySelector('[data-testid="list-view-items"]')
```

We can do this, but I would rather ignore the code smell. The ternary is more readable.